frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
+                }
+            },
+            "Action": [
+                "sagemaker:DeleteNotebookInstance",
+                "sagemaker:DescribeNotebookInstance",
+                "sagemaker:StopNotebookInstance"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:DeleteNotebookInstanceLifecycleConfig",
+                "sagemaker:DescribeNotebookInstanceLifecycleConfig"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance-lifecycle-config/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:ListNotebookInstanceLifecycleConfigs",
+                "sagemaker:ListNotebookInstances"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": "lambda:InvokeFunction",
+            "Resource": "arn:{AWS_PARTITION}:lambda:{AWS_REGION}:{AWS_ACCOUNT}:function:mls-lambda-*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:ListClusters",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:DescribeCluster",
+                "elasticmapreduce:ListInstances",
+                "elasticmapreduce:TerminateJobFlows",
+                "elasticmapreduce:SetTerminationProtection"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:elasticmapreduce:{AWS_REGION}:{AWS_ACCOUNT}:cluster/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "translate:StopTextTranslationJob",
+                "translate:ListTextTranslationJobs",
+                "translate:DescribeTextTranslationJob",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "StringEqualsIgnoreCase": {
+                    "iam:ResourceTag/system": "MLSpace"
+                }
+            },
+            "Action": [
+                "iam:DetachRolePolicy",
+                "iam:DeleteRole",
+                "iam:DeleteRolePolicy",
+            ],
+            "Resource": "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:role/MLSpace*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:ListRoles",
+                "iam:ListEntitiesForPolicy",
+                "iam:ListPolicyVersions",
+                "iam:ListAttachedRolePolicies",
+                "iam:GetRole",
+                "iam:GetPolicy"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:DeletePolicy",
```

Comment:
The app config update lambda needs the following permissions:
```
'iam:CreatePolicy',
'iam:CreatePolicyVersion',
'iam:TagPolicy',
```

It needs to be able to modify dynamic roles. Again this is something that other lambdas working with dynamic roles will need and something I think should be standard for the System role

Reply:
Good call out thanks